src/radCommonAutoComplete.js

```diff
@@ -1,16 +1,12 @@
-import jQuery from 'jquery';
```

Comment:
I can't exactly remember why we did this before, where we used "jQuery" instead of "$" maybe it was conflict with the global window, but assuming you tested this, but just wanted to make sure nothing else is going on.